gradle.properties

```diff
@@ -9,7 +9,7 @@ android.useAndroidX=true
 android.nonTransitiveRClass=true
 
 # Gradle
-org.gradle.jvmargs=-Xmx8G -Dfile.encoding\=UTF-8 -XX:MaxMetaspaceSize=512m -Dkotlin.daemon.jvm.options\=-Xmx8G,-XX:MaxMetaspaceSize=512m
+org.gradle.jvmargs=-Xmx8G -Dfile.encoding\=UTF-8 -XX:MaxMetaspaceSize=4G -Dkotlin.daemon.jvm.options\=-Xmx8G,-XX:MaxMetaspaceSize=4G
```

Comment:
If you're having problems on CI this is overridden [there](https://github.com/splendo/kaluga/blob/develop/.github/workflows/publish.yaml#L27), so you might have to update this atop the workflow files as well. Although indeed this default is very low. 4GB of just code (metaspace) is pretty be massive though.

Reply:
Yeah reverting. Dokka has a memory leak, so had to increase it for that, but no need to propagate that